### ISSUE TITLE
template template's 

### ISSUE THREAD

**Issue by nkdmSum** [jeff-hykin/better-cpp-syntax#537]
- [x] This problem exists even with the setting `"C_Cpp.enhancedColorization": "Disabled"`
- [ ] This bug exists for C
- [x] This bug exists for C++
- [ ] This bug exists for Objective-C
- [ ] This bug exists for Objective-C++

## The code with a problem is:
```
template<typename K, typename V, template<typename> typename C>
```
## It looks like:
<img width="723" alt="Снимок экрана 2021-02-13 в 12 00 48" src="https://user-images.githubusercontent.com/51021475/107848368-22027900-6df3-11eb-854c-e87eec86572e.png">

## It should look like:
Both the inner `template` keyword, as well as the `typename` keyword should be highlighted

**Comment by j-cortial** [comment]
This issue should be solved now.

**Comment by jeff-hykin** [comment]
published today with `v1.16.3`
